nix/pkgs/web-ghc/default.nix

```diff
@@ -11,7 +11,9 @@ let
   ] ++ (extraPackagesFun ps));
 
   runtimeGhcWrapped = writeShellScriptBin "runghc" ''
-    export PATH=${lib.makeBinPath [ bubblewrap runtimeGhc util-linux ]}
+    export PATH=$PATH:${lib.makeBinPath [ bubblewrap runtimeGhc util-linux ]}
+    echo $PATH 
+    echo AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
```

Comment:
Is this debugging info that should have been removed?

Reply:
This file is getting deleted in the next PR